python/ray/_private/ray_constants.py

```diff
@@ -485,3 +485,6 @@ def gcs_actor_scheduling_enabled():
 
 
 RAY_NODE_IP_FILENAME = "node_ip_address.json"
+
+RAY_ENABLE_MIG_DETECTION_ENV_VAR = "RAY_ENABLE_MIG_DETECTION"
+RAY_ENABLE_MIG_DETECTION = env_bool(RAY_ENABLE_MIG_DETECTION_ENV_VAR, False)
```

Comment:
Should the default value be true? If I understand correctly, when a GPU is splited to MIG devices, the original id is useless?

Reply:
I thought we hide it behind feature flag so it just detect normal GPU so no risky change. if user want to start cluster with MIGs, then they can enable the flag by themselves.
Also for second question, yes original id is ignored